### PR TITLE
Enforce LF.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Enforce Unix newlines
+* text=auto eol=lf
```
Comment (styfle) [inline .gitattributes]: What's the advtange of this vs prettier?

**Comment by XhmikosR** [comment]
It's unrelated to prettier. This is on the git level. On Windows with
autocrlf enabled which is the default, one ends up with CRLF. This ensures
that only LF is used.

**Comment by styfle** [comment]
In addition to this change, I think it would be best to add `"endOfLine": "lf"` to `.prettierrc.json` according to the docs here: https://prettier.io/docs/en/options.html

**Comment by XhmikosR** [comment]
You can do that later, sure. If you had this enabled, it would fail for Windows which is what this patch fixes. Consistent line endings everywhere.

**Pull request merged**